global/config/nginx/certs/default.crt

```diff
@@ -0,0 +1,17 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Do we need these cert files? I don't see we use it in the global docker-compose file, so we can probably remove it, right?

Reply:
There are cases that I ran into where https wouldn't be listening because of the missing cert. This would prevent you from getting an error message that the site isn't available or running

Reply:
@dustinrue in this case we need to add the folder with these certificates to the nginx-proxy volumes list in the docker-compose file, don't we?